tests/data/onboarding/index.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  appTitle: "Ledger Live",
+  duskColor: "#182532",
+  darkColor: "#1c1d1f",
+  lightColor: "#ffffff",
+  welcomeTitle: "Welcome to Ledger Live",
+  welcomeDesc:
+    "Let’s start by choosing how Ledger Live looks to you. You can change this later at any time in your settings.",
+  getStartedTitle: "Get started with your Ledger device",
+  selectDeviceTitle: "Select your device",
+  choosePinTitle: "Choose your PIN code",
+  securityTitle: "Security checklist",
+  securityDesc: "Before getting started, please confirm:",
+  securityDesc2: "Before continuing, please complete the security checklist", // FIXME: Wording duplicates
+  saveSeedTitle: "Write down your recovery phrase",
+  enterSeedTitle: "Enter your recovery phrase",
+  genuineModalTitle: "Genuine check",
+  genuinePinErrorTitle: "Didn't choose your own PIN code?",
+  genuinePinErrorDesc:
+    "Never use a device supplied with a PIN code. Please contact us when in doubt.",
+  genuineSeedErrorTitle: "Didn't save your recovery phrase by yourself?",
+  genuineSeedErrorDesc:
+    "Only save a recovery phrase that was displayed on your device and to you alone. Please contact us when in doubt.",
+  noDeviceTitle: "Don't have a Ledger device?",
```

Comment:
note for later. we shouldn't maintain this but dynamically reverse it with `i18n`. this will be a dev task.